projects/packages/action-bar/src/js/utils.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Add class to element
+ *
+ * @param {Element} el - element
+ * @param {string} className - class to add
+ */
+export function _classAdd( el, className ) {
+	if ( ! _exists( el ) || ! className ) {
+		return;
+	}
+	if ( el ) {
+		if ( _classHas( el, className ) ) {
+			return;
+		}
+		if ( el.classList ) {
+			el.classList.add( className );
+		} else {
+			el.className += ' ' + className;
+		}
+	}
+}
+
+/**
+ * Check if class exists
+ *
+ * @param {Element} el - element
+ * @param {string} className - class to check for
+ * @returns {boolean} true if class exists
+ */
+export function _classHas( el, className ) {
+	if ( ! el || ! className ) {
+		return false;
+	}
+	return el.classList.contains( className );
+}
+
+/**
+ * Remove class from element
+ *
+ * @param {Element} el - element
+ * @param {string} className - class to remove
+ * @returns {boolean} status
+ */
+export function _classRemove( el, className ) {
+	if ( ! _exists( el ) || ! className ) {
+		return false;
+	}
+	if ( el.classList ) {
+		return el.classList.remove( className );
+	}
+	return ( el.className = el.className.replace(
+		new RegExp( '(^|\\b)' + className.split( ' ' ).join( '|' ) + '(\\b|$)', 'gi' ),
+		' '
+	) );
+}
```

Comment:
I don't think we need these for manipulating classes. classList.add, remove, and contains seems to have good enough browser support now.

Reply:
I simplified that 👍